entrypoint.sh

```diff
@@ -68,9 +68,32 @@ function dev_part {
   fi
 }
 
+function osd_trying_to_determine_scenario {
+  if [[ ! -d /var/lib/ceph/osd || -n "$(find /var/lib/ceph/osd -prune -empty)" ]]; then
```

Comment:
this changed recently, we need to backport https://github.com/ceph/ceph-docker/pull/335